problems/countNumbers.js

```diff
@@ -9,9 +9,17 @@
  * returns { 1:3, 2:2, 3:1, 4:1 }
  */
 
-function countNumbers() {
- 
-}
-
+function countNumbers(arr) {
+        let counter = {};
+        for(let i = 0; i < arr.length; i++) {
+            const el = arr[i];
+            if(counter[el]) {
+                counter[el] += 1;
+            } else {
+                counter[el] = 1;
+            }
+        }
+        return counter; 
+    }
```

Comment:
Great job on getting through this. This is a really common pattern that you'll see in questions. Make sure to have it memorized. I see you got this from the notes, that's okay. Utilizing the tools online / from notes is also an important skill for when you're on the job.